cassandane/Cassandane/Util/Words.pm

```diff
@@ -84,4 +85,16 @@ sub random_word
     return $remaining[int(rand(scalar @remaining))];
 }
 
+sub random_words
+{
+    my ($count) = @_;
+    my @random_words;
+
+    while ($count-- > 0) {
+        push @random_words, random_word();
+    }
+
+    return wantarray ? @random_words : join($", @random_words);
```

Comment:
Hmm.  In my mind interpolation is relatively slow, so if there's a specific way to do something, it's better to do that rather than interpolate.  But I don't know where I got that from, maybe it's rubbish anyway!

The irony is, if I'd been reading code with the array interpolated, I wouldn't have known off the top of my head how the words would be separated, because I didn't know about `$"` until I was reading the docs to write this line.  So in a way, this line is a note to my future self to remember how arrays interpolate. :)

Reply:
Under the hood they are exactly the same:
```
alh@lockedtomb:~$ perl -MO=Concise -e 'my $x = "@list"'
a  <@> leave[1 ref] vKP/REFC ->(end)
1     <0> enter v ->2
2     <;> nextstate(main 1 -e:1) v:{ ->3
9     <2> sassign vKS/2 ->a
7        <@> join[t5] sK/2 ->8
3           <0> pushmark s ->4
-           <1> ex-rv2sv sK/1 ->5
4              <#> gvsv[*"] s ->5
6           <1> rv2av[t4] lK/1 ->7
5              <#> gv[*list] s ->6
8        <0> padsv[$x:1,2] sRM*/LVINTRO ->9
-e syntax OK
```
```
alh@lockedtomb:~$ perl -MO=Concise -e 'my $x = join($", @list)'
a  <@> leave[1 ref] vKP/REFC ->(end)
1     <0> enter v ->2
2     <;> nextstate(main 1 -e:1) v:{ ->3
9     <2> sassign vKS/2 ->a
7        <@> join[t5] sK/2 ->8
3           <0> pushmark s ->4
-           <1> ex-rv2sv sK/1 ->5
4              <#> gvsv[*"] s ->5
6           <1> rv2av[t4] lK/1 ->7
5              <#> gv[*list] s ->6
8        <0> padsv[$x:1,2] sRM*/LVINTRO ->9
-e syntax OK
```

I had to look up `$"`, I was fairly sure what it was going to be but had to be sure heh. It's fine.